src/calendar.h

```diff
@@ -205,7 +213,7 @@ class time_duration
         }
         template<typename T>
         static constexpr time_duration from_minutes( const T m ) {
-            return from_turns( m * 60 );
+            return from_turns( m * 60 / seconds_per_turn );
```

Comment:
Ideally this should defer to `from_seconds`, to match other `from_*` methods
```suggestion
            return from_seconds( m * 60 );
```

Either way, `from_seconds` method needs the update too